docs/cloud/dbaas/firewalls.md

```diff
@@ -0,0 +1,99 @@
+# Using Firewalls in Pukki
+
+!!! error "Closed Beta"
+    Pukki DBaaS is in closed beta. This means that the service is probably not suitable for most users
+    and there might be breaking changes. If you are still interested in using the service you can
+    [contact us](../../support/contact.md) to see if the service would be suitable for you.
+
+
+All database instances have their own firewalls. Users are responsible for making sure that the firewall rules are strict. The firewall rules should only be open to those IP-addresses that is needed. Relaxed firewall rules are probably some of the largest security risks and you need to take it seriously. Even if you don't have any "secret" data in your database, you are not allowed to have it open to the world. If you want to share your data, you should do it through a proxy or other services that might use the database as a backend. Leaving a database port open on the internet is an sure-fire way to attract malicious actors to target your database.
+
+## How to manage firewalls
+
+You can change the firewalls from the [web interface](https://pukki.dbaas.csc.fi) on existing
+instances by pressing the "Update Instance".
+
+With the [openstack CLI](cli.md) tool you can use the `opsenstack database instance update --help` command.
+Note that the command override the existing firewalls rules which means that you need to set all
+the firewall opening each time you update the firewalls for an instance with `--allowed-cidr` flag.
+
+## Single IP or subnet
+
+It is possible to allowed a single IP by adding the "CIDR notion" `/32` suffix to each IP. It is
```

Comment:
Since can't come up with other improvements - I'll nitpick the grammary: "It is possible to allow..." or "It is possible to have a single IP allowed..."

Reply:
Changed that paragraph a bit.